Exiled.Events/Patches/Generic/TargetOffset.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace Exiled.Events.Patches.Generic
 {
+#pragma warning disable IDE0060 // Supprimer le paramètre inutilisé
```

Comment:
What this warning about? Seems to be IDE specific so idk if we really need it

Reply:
I'm kinda late but it says "Remove unused parameter". so it removes parameters being flagged when they aren't used and they do not start with the _ symbol